extensions/kafka.md

```diff
@@ -254,7 +250,7 @@ public class KafkaEventConsumptionConfiguration {
 }
 ```
 
-Note that as with any tracking event processor, the progress on the event stream is stored in a `TrackingToken`. Using the `StreamableKafkaMessageSource` means a `KafkaTrackingToken` containing topic-partition to offset pairs is stored in the `TokenStore`.
+Note that as with any tracking event processor, the progress on the event stream is stored in a `TrackingToken`. Using the `StreamableKafkaMessageSource` means a `KafkaTrackingToken` containing topic-partition to offset pairs is stored in the `TokenStore`. From version 4.6.0 there will also be a `TokenStore` implementation available based on Kafka, from this extension.
```

Comment:
Defining the specifics on 4.6.0 shouldn't be necessary.
The `master` branch will become the `4.6.0` selected Reference Guide page once we've made a `4.6.0` branch.
Differently put, if a user has selected the guide for version 4.6.0, the fact there's a `KafkaTokenStore` is simply a given.

Reply:
Could not resist the temptation. I guess this should target 4.5 then. Made the changes accordingly.